kubejs/server_scripts/mods/optionalCompats/advanced_peripherals.js

```diff
@@ -0,0 +1,142 @@
+            C: `${modid}:peripheral_casing`,
+            A: "minecraft:paper",
+            S: "gtceu:mv_sensor"
+        })
+
+        replaceRecipe("energy_detector", [
+            'C C',
+            'TPT',
+            'C C'
+        ], {
+            C: "gtceu:copper_single_cable",
+            T: "gtceu:mv_transformer_1a",
+            P: `${modid}:peripheral_casing`
+        })
+
+        event.remove({ id: `${modid}:environment_detector` })
+        const envDetectorRecipe = (plate) => {
+            event.shaped(`${modid}:environment_detector`, [
+                'PSP',
+                'SCS',
+                'DSD'
+            ], {
+                P: plate,
+                S: "gtceu:lv_sensor",
+                C: `${modid}:peripheral_casing`,
+                D: "minecraft:dirt"
+            })
+        }
+        envDetectorRecipe("gtceu:nether_quartz_plate")
+        envDetectorRecipe("gtceu:certus_quartz_plate")
+        envDetectorRecipe("gtceu:quartzite_plate")
+
+        replaceRecipe("player_detector", [
+            'OEO',
+            'PCP',
+            'MSM'
+        ], {
+            O: "minecraft:observer",
+            E: "gtceu:mv_emitter",
+            P: "minecraft:ender_pearl",
+            C: `${modid}:peripheral_casing`,
+            M: "minecraft:smooth_stone",
+            S: "gtceu:mv_sensor"
+        })
+
+        replaceRecipe("inventory_manager", [
+            'ETE',
+            'RPR',
+            'ECE'
+        ], {
+            E: "minecraft:ender_pearl",
+            T: "minecraft:crafting_table",
+            R: "gtceu:mv_robot_arm",
+            P: `${modid}:peripheral_casing`,
+            C: "minecraft:chest"
+        })
+
+        replaceRecipe("block_reader", [
+            'PSP',
+            'ECM',
+            'PSP'
+        ], {
+            P: "gtceu:steel_plate",
+            S: "gtceu:steel_screw",
+            E: "gtceu:mv_emitter",
+            C: `${modid}:peripheral_casing`,
+            M: "gtceu:mv_sensor"
+        })
+
+        replaceRecipe("nbt_storage", [
+            'PMP',
+            'SRS',
+            'PCP'
+        ], {
+            P: "gtceu:steel_plate",
+            M: "computercraft:wired_modem",
+            S: "#gtceu:circuits/mv",
+            R: `${modid}:peripheral_casing`,
+            C: "minecraft:chest"
+        })
+
+        replaceRecipe("geo_scanner", [
+            'SPS',
+            'CRC',
+            'SDS'
+        ], {
+            S: "gtceu:steel_screw",
+            P: "gtceu:prospector.lv",
+            C: "#gtceu:circuits/mv",
+            R: `${modid}:peripheral_casing`,
+            D: "minecraft:diamond"
+        })
+
+        replaceRecipe("me_bridge", [
+            'GIG',
+            'SCS',
+            'GTG'
+        ], {
+            G: "ae2:fluix_glass_cable",
+            I: "ae2:interface",
+            S: "#gtceu:circuits/mv",
+            C: `${modid}:peripheral_casing`,
+            T: "ae2:pattern_provider"
+        })
+    })
```

Comment:
> That suggests that it should be further in the progression than MV.

I would disagree, export bus with a craft card on a drawer makes a great requester, which me and my friend used to use in nomifactory (worked very well there) and are using here because ME requesters are buggy (putting them behind a toggle bus makes them refuse to work, cannot input fractional number into fluid request), so if anything the requesters should go down in tier (they're fine where they are for their "convenience" though (if they worked)).

> perhaps EV

Another point here would be that we used and are using the ME bridge for very simple functionality like checking how much energy a network consumes, which, again, imho, should not require EV components.

And in the end of the day, ME requester functionality isn't something game breaking, and considering that implementing it in cc would require quite a bit of effort, I would say MV is a nice tier for it.

Reply:
> And in the end of the day, ME requester functionality isn't something game breaking, and considering that implementing it in cc would require quite a bit of effort, I would say MV is a nice tier for it.

While it may not be game breaking, the requester was recently pushed back from HV to LuV. It seems like providing equivalent functionality at MV in a compat mod doesn't make sense. I do agree that it requires a bit of effort to implement using CC (though I think you can find scripts online that save a bunch of that effort), which is why I suggested EV (which is two tier before you can get ME requesters).

(Note than I am not one of the developers of this pack, so can't make the final call.)

Reply:
Wow. LuV for requesters? Yeah, no, I think I'll take my chances, leave as is, and see what main devs say here.

(Thanks for the comments still)